content/javascript/concepts/arrays/terms/findLast/findLast.md

```diff
@@ -0,0 +1,81 @@
+---
+Title: '.findLast()'
+Description: 'Returns the first value found in the array by starting the search from the end of the array to the beginning of the array.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Arrays'
+  - 'Methods'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `.findLast()` array method returns the first value found in the array by starting the search from the end of the array to the beginning of the array.
+
+## Syntax
+
+```js
+array.findLast((element, index) => {...});
+```
+
+The function can be invoked with two arguments:
+
+- `element`: The current element we are iterating through.
+- `index` (optional): The index of the array element.
+
+## Example
+
+Find the value 200 in the array:
+
+```js
+const numbers = [10, 42, 53, 12, 45, 66, 33, 6, 10000, 200, 10];
+
+const findNumber = numbers.findLast((value) => {
+  return value === 200;
+});
+
+console.log(findNumber);
+```
+
+This results in the following output:
+
+```shell
+200
+```
+
+## Codebyte Example
```

Comment:
It looks as though the docs js compiler doesn't currently support this method. Please remove this example

Reply:
Are we open to creating a new issue for a Codebyte for this method? @caupolicandiaz @SSwiniarski

Reply:
@codewithfan, that won't do much since this isn't going to be resolved through the Docs repo. This has to do with Codecademy's infrastructure and the version of JavaScript they support.